flink-table/flink-table-common/src/main/java/org/apache/flink/table/api/dataview/DataView.java

```diff
@@ -19,18 +19,20 @@
 package org.apache.flink.table.api.dataview;
 
 import org.apache.flink.annotation.PublicEvolving;
-
-import java.io.Serializable;
+import org.apache.flink.table.functions.ImperativeAggregateFunction;
 
 /**
  * A {@link DataView} is a collection type that can be used in the accumulator of an
- * {@link org.apache.flink.table.functions.AggregateFunction}.
+ * {@link ImperativeAggregateFunction}.
```

Comment:
Is the `AggregateFunction` no longer valid here?

Reply:
I found out the answer myself. I was not aware the `ImperativeAggregateFunction` is more generic. Both `AggregateFunction` & `TableAggregateFunction` extend from it.
Thought it is the other way around.